trollmoves/server.py

```diff
@@ -129,28 +129,28 @@ def pong(self, message):
     def push(self, message):
         """Reply to push request."""
         new_msg = self._move_files(message)
-        if new_msg is None:
+        if new_msg.type is not 'err':
+            _destination = clean_url(new_msg.data['destination'])
```

Comment:
Isn't the destination already cleaned in `_move_file()`? Or actually, maybe safer to do the cleaning here if the `err` message also has some credentials or such that needs to be cleaned.